core/src/test/scala/unit/kafka/log/LogCleanerLagIntegrationTest.scala

```diff
@@ -44,18 +44,19 @@ class LogCleanerLagIntegrationTest(compressionCodecName: String) extends Logging
 
   val time = new MockTime(1400000000000L, 1000L)  // Tue May 13 16:53:20 UTC 2014 for `currentTimeMs`
   val cleanerBackOffMs = 200L
-  val segmentSize = 100
+  val segmentSize = 512
```

Comment:
Note I couldn't find a good reason for why the segment size was so low, but it results in only one message per segment. This seems like an unrealistic scenario, so I bumped it a little bit to ensure we're getting multiple messages per segment. As a bonus, this makes test run a little faster.